jatindhankhar/memory_game_hack.js

```diff
@@ -1,6 +1,7 @@
 classes = ['fa-cube', 'fa-diamond', 'fa-leaf', 'fa-bolt', 'fa-bomb', 'fa-anchor', 'fa fa-paper-plane-o', 'fa fa-bicycle']
 
-for(var i = 0 ; i < classes.length ; i++){
-	var x = document.getElementsByClassName(classes[i]);
-	for(var j=0; j <x.length ; j++){x[j].parentElement.click();}
+for(let targetClass of classes){
+	
```

Comment:
Extra `\n`?

Reply:
Might have missed it. Didn't pass it through a code formatter.

Reply:
Fixed.